ArduPlane/control_modes.cpp

```diff
@@ -161,9 +161,19 @@ void Plane::reset_control_switch()
 void Plane::autotune_start(void)
 {
     gcs().send_text(MAV_SEVERITY_INFO, "Started autotune");
-    rollController.autotune_start();
-    pitchController.autotune_start();
-    yawController.autotune_start();
+    uint8_t bitmask = g2.axis_bitmask;
+    if (bitmask & AUTOTUNE_AXIS_BITMASK_ROLL) { 
+        rollController.autotune_start();
+        gcs().send_text(MAV_SEVERITY_INFO, "Starting Roll");
+    }
+    if (bitmask & AUTOTUNE_AXIS_BITMASK_PITCH) { 
+        pitchController.autotune_start();
+        gcs().send_text(MAV_SEVERITY_INFO, "Starting Pitch");
+    }
+    if (bitmask & AUTOTUNE_AXIS_BITMASK_YAW) { 
+        yawController.autotune_start();
+        gcs().send_text(MAV_SEVERITY_INFO, "Starting Yaw");
+    }
```

Comment:
By default this is now going to print four lines. We should be able to get them on a single line. "Starting Roll Pitch Yaw" ect.

Reply:
Something like:
```
    const bool tune_roll = g2.axis_bitmask.get() & AUTOTUNE_AXIS_BITMASK_ROLL;
    const bool tune_pitch = g2.axis_bitmask.get() & AUTOTUNE_AXIS_BITMASK_PITCH;
    const bool tune_yaw = g2.axis_bitmask.get() & AUTOTUNE_AXIS_BITMASK_YAW;
    if (tune_roll) { 
        rollController.autotune_start();
    }
    if (tune_pitch) { 
        pitchController.autotune_start();
    }
    if (tune_yaw) { 
        yawController.autotune_start();
    }
    gcs().send_text(MAV_SEVERITY_INFO, "Started autotune %s%s%s", tune_roll?"roll ":"", tune_pitch?"pitch ":"", tune_yaw?"yaw":"")
```

Reply:
yep...like that better....changed to a slight variant